css/custom-styles/_site-header.scss

```diff
@@ -8,12 +8,20 @@
 
 $site-navbar-height: 10;
 $site-secondary-nav-input-height: 38px;
+$site-inner-nav-height: 6;
 
 // Remove max-width from header containers
 // See https://github.com/uswds/uswds/issues/4851
 .site-header__navbar,
 .site-nav__inner {
-  max-width: none !important;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: units("widescreen");
+  position: relative;
```

Comment:
Center these elements and constrain them to `widescreen` like other full-width page element